src/DebugChannel.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+import path from "path";
+
+export class DebugChannel {
+  constructor(fs: any) {
+    let configFilePath = path.join(__dirname, "../src/debugger/config.json");
+    let config = JSON.parse(fs.readFileSync(configFilePath, "utf8").toString());
+    this.inFilePath = path.join(__dirname, "../", config.files.proxy2debugger);
+    this.outFilePath = path.join(__dirname, "../", config.files.debugger2proxy);
+    this.fs = fs;
+  }
+  inFilePath: string;
+  outFilePath: string;
+  fs: any;
+
+  readIn(): string {
+    let contents = this.fs.readFileSync(this.inFilePath, "utf8");
+    return contents;
+  }
+
+  writeOut(contents: string): void {
+    this.fs.writeFileSync(this.outFilePath, contents);
```

Comment:
Note that every time this is called, the entire contents of the file is replaced. This implies that the debugger UI and the debugger code in Prepack have to maintain a strict and synchronous request response protocol. Please add some documentation here to inform the future reader of these assumptions.